third_party/terraform/resources/resource_logging_sink.go

```diff
@@ -56,21 +73,39 @@ func flattenResourceLoggingSink(d *schema.ResourceData, sink *logging.LogSink) {
 	d.Set("writer_identity", sink.WriterIdentity)
 }
 
-func expandResourceLoggingSinkForUpdate(d *schema.ResourceData) *logging.LogSink {
+func expandResourceLoggingSinkForUpdate(d *schema.ResourceData) (*logging.LogSink, string) {
```

Comment:
nit: A named return here would be helpful to future developers.

Reply:
Good point. Fixed to use named returns just to express meaning: 63bc19c

Reply:
Made some mistake...fixed 70cc776